lib/json/static.nit

```diff
@@ -31,6 +31,9 @@ private import json_lexer
 interface Jsonable
 	# Encode `self` in JSON.
 	#
+	# This is a recursive method which can be refined by any instances.
```

Comment:
I wouldn't mind the `subclasses` part, but not the `must`. The abstract part already tells that direct instantiable classes must redef, but not other subclasses.

And please don't bold at me, I can spot the difference between 2 sentences.